sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetWriteSupport.scala

```diff
@@ -73,6 +73,9 @@ class ParquetWriteSupport extends WriteSupport[InternalRow] with Logging {
   // Reusable byte array used to write timestamps as Parquet INT96 values
   private val timestampBuffer = new Array[Byte](12)
 
+  // Reusable byte array used to write intervals as Parquet FIXED_LEN_BYTE_ARRAY values
+  private val intervalBuffer = new Array[Byte](12)
```

Comment:
Not that it matters much, but do you need a separate reusable byte buffer? I guess that makes sure we don't accidentally change one and affect the other, but it wouldn't change?

Reply:
I will combine the buffers.